### DISCUSSION TITLE
Update Twitter Icon - X

### DISCUSSION THREAD

**Discussion by ghost** [ant-design/ant-design#43805] [Ideas]
Will the Twitter icon be updated to address the new X branding they are currently using?

**Comment by Wxh16144** [comment]
### 6
> a number that is used by the Chinese to express surprise, more often than not, to marvel at something.

Related news reports:
1. https://www.nbcnews.com/news/us-news/twitter-rebrands-x-elon-musk-loses-iconic-bird-logo-rcna95880
2. https://twitter.com/elonmusk/status/1683378289031761920
3. https://www.wsj.com/video/elon-musk-rolls-out-new-x-logo-as-twitter-rebrands/0F91130A-ED9D-497B-A282-F26CCCF5DD91.html

**Comment by MadCcc** [comment]
Since Twitter will be renamed to X, there will be both of them.

**Comment by abc1196** [comment]
Any updates on this new icon?

**Comment by kevingio-julo** [comment]
any update on this icon?
